### DISCUSSION TITLE
Conflicting UI for setting the default option for a dropdown field within a table

### DISCUSSION THREAD

**Discussion by missmatsuko** [craftcms/cms#8981] [Ideas]
### Description
There appear to be 2 places to set the default option for a dropdown within a table. 

One is within the dropdown settings where you set the options, and the other is in the table display where all the other columns' defaults are set. The latter makes more sense to me. 

![Screenshot of dropdown options.](https://user-images.githubusercontent.com/17884198/102725749-eef9a880-42cd-11eb-8e5d-0f4ce2913e72.png)
![Screenshot of table defaults.](https://user-images.githubusercontent.com/17884198/102725750-ef923f00-42cd-11eb-9701-e6fc906f57eb.png)

When a default option is selected using either method, it's not synced with the other.

There also doesn't seem to be a way to set the dropdown to have no default option using the defaults table UI.

### Additional info

- Craft version: 3.5.17.1

**Comment by brandonkelly** [comment]
They’re slightly different: The “Default Values” setting defines the initial state of the field for new elements. It does _not_ have any effect on new rows that are added while editing an element.

The “Default?” checkbox within the Dropdown Options setting defines the default option to be selected for new rows added by clicking the “Add a row” button while editing an element.

**Comment by missmatsuko** [comment]
Hmm, okay... but why does only the dropdown field has a "Default?" setting?

**Comment by brandonkelly** [comment]
To determine which option should be selected by default when new rows are added _by the authors_.

**Comment by missmatsuko** [comment]
Yes, I understand the difference between the "Default Values" and "Default?" settings now (although I'm not sure if it's clear from the UI), but I don't get why only dropdown has the "Default?" feature? 

For example, we can't set defaults new rows for checkboxes (always unchecked for new rows) and color (always blank for new rows), right?

Also curious if it's common to want default row(s) that are different from the default for new rows.

**Comment by brandonkelly** [comment]
True. Dropdown columns are a bit of an exception as they are the only ones with an extended settings modal, as it was a necessity in order to define the available options.

I suppose it would make more sense for Table fields to have a “Default Row Values” setting, which allows developers to set the default values for _all_ columns for new author-added rows.

It would be awkward to have that _and_ the current “Default Values” setting; maybe we just drop the ability to add multiple rows to that setting, and add a new “Default Rows” setting that defines the number of initial rows that should be created for new elements by default – all based on the “Default Row Values” setting.

**Comment by missmatsuko** [comment]
Yep, that makes sense to me!
